src/hotspot/share/prims/jvmtiEnvBase.cpp

```diff
@@ -2359,6 +2407,21 @@ SetFramePopClosure::doit(Thread *target, bool self) {
   _result = ((JvmtiEnvBase*)_env)->set_frame_pop(_state, jvf, _depth);
 }
 
+void
+SetFramePopClosure::do_vthread(Handle target_h) {
+  Thread* current = Thread::current();
+
+  if (!JvmtiEnvBase::is_vthread_alive(_target_h())) {
+    return; // JVMTI_ERROR_THREAD_NOT_ALIVE (default)
+  }
```

Comment:
Don't we have this check already in JvmtiHandshake::execute()? Same with the other converted functions.

Reply:
Good suggestion, thanks.
I'm a little bit paranoid about terminated vthreads. :)
Will try to get rid and retest all tiers.

Reply:
I've removed the extra checks for `JvmtiEnvBase::is_vthread_alive()`.